src/Modules/Finance/Header/Header.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { TargetSum } from './TargetSum/TargetSum';
+import { Total } from './Total/Total';
+import { Transaction } from './Transaction/Transaction';
+
+import styles from './Header.module.css';
+
+type IHeaderProps = {

```

Comment:
In case you create a type start he name with a capital `T` letter. You should use capital `I` if you create an interface.

Reply:
Done